python/ray/serve/api.py

```diff
@@ -523,7 +435,9 @@ def run(
             to execute the serve DAG.
     """
 
-    client = start(detached=True, http_options={"host": host, "port": port})
+    client = _private_api.serve_start(
+        detached=True, http_options={"host": host, "port": port}
+    )
```

Comment:
What is the behavior when `serve.run` is called twice without shutting down Serve? In this case, the host and port may not be respected. At the very least we should print a warning.

Reply:
Added warning.